Packs/CommonScripts/.secrets-ignore

```diff
@@ -4,4 +4,6 @@ https://typora.io
 https://expired.badssl.com
 randomName@randomDomain.com
 @pytest.mark.parametrize
-AB@1Ab.Vt
\ No newline at end of file
+AB@1Ab.Vt
+multipart_sigened
+/utf_8_with_bom.eml
```

Comment:
run the `demisto-sdk secrets` locally with the `-ie` flag and see if these secrets are really necessary

Reply:
done